Framework/Geometry/src/Instrument/ComponentInfoBankHelpers.cpp

```diff
@@ -32,6 +32,48 @@ bool isDetectorFixedInBank(const ComponentInfo &compInfo,
   return false;
 }
 
+bool isAnyBank(const ComponentInfo &compInfo, const size_t &idx) {
```

Comment:
the function itself doesn't implicitly check that saveInstrument can be saved (it always should), isAnyNXdetector?

Reply:
I would definitely drop `Any` from the name, i'm also reluctant to refer to Nexus in a part of the codebase that shouldn't be Nexus aware. 

I prefer the name I have specified above for these reasons. `isSaveableBank`. Comments needed of course since this there are Assembly types that are being skipped over and users of these function need to understand what/why that is.

Reply:
done